src/ast/cairoNodes/tempvar.ts

```diff
@@ -0,0 +1,13 @@
+import { Statement, StructuredDocumentation } from 'solc-typed-ast';
+
+export class CairoTempVar extends Statement {
```

Comment:
I think appending `Statement` to the class name would be good for readability purposes. When reading later the code it's clear we are creating a Statement. Also this way, we follow the same pattern as solc-typed-ast nodes